start.py

```diff
@@ -721,10 +721,10 @@ def _configure_sdc(deployment, cluster, is_kerberos_enabled):
                                                   'type': 'STREAMSETS',
                                                   'displayName': 'StreamSets',
                                                   'roles': [datacollector_role]}])
-    # When running an application on YARN, the Spark executor requires access to the spark-submit script located in
-    # the Spark installation directory. Default is directory specified by SPARK_HOME environment variable.
-    # Hence SPARK_HOME environment variable must be set before starting StreamSets Data Collector.
-    configs = {'sdc-env.sh_role_safety_valve': 'export SPARK_HOME=/opt/cloudera/parcels/CDH/lib/spark'}
+    # When running an application  with Spark2, the following
+    # environment variables must be set before starting StreamSets Data Collector.
+    config_value = 'export SPARK_SUBMIT_YARN_COMMAND=/usr/bin/spark2-submit\nexport SPARK_KAFKA_VERSION=0.10'
```

Comment:
Is it really cool like this to hardcode this `SPARK_KAFKA_VERSION` thing or will it potentially bite us at some point?